tests/snapshots/tests.contrib.azure_functions.test_azure_functions_snapshot.test_service_bus_distributed_tracing[enabled].json

```diff
@@ -0,0 +1,126 @@
+[[
+  {
+    "name": "azure.functions.invoke",
+    "service": "test-func",
+    "resource": "POST /api/httppostrootservicebus",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "serverless",
+    "meta": {
+      "_dd.p.dm": "-0",
+      "_dd.p.tid": "684c654000000000",
+      "aas.function.name": "http_post_root_servicebus",
+      "aas.function.trigger": "Http",
+      "component": "azure_functions",
+      "http.method": "POST",
+      "http.route": "/api/httppostrootservicebus",
+      "http.status_code": "200",
+      "http.url": "http://0.0.0.0:7071/api/httppostrootservicebus",
+      "http.useragent": "python-httpx/x.xx.x",
+      "language": "python",
+      "runtime-id": "3e1d1c4aeb5940e298b66654dc72a8ee",
+      "span.kind": "server"
+    },
+    "metrics": {
+      "_dd.top_level": 1,
+      "_dd.tracer_kr": 1.0,
+      "_sampling_priority_v1": 1,
+      "process_id": 73382
+    },
+    "duration": 2035868084,
+    "start": 1749837120334504592
+  },
+     {
+       "name": "azure.servicebus.produce",
+       "service": "azure_servicebus",
+       "resource": "queue.1",
+       "trace_id": 0,
+       "span_id": 2,
+       "parent_id": 1,
+       "type": "worker",
+       "meta": {
+         "_dd.base_service": "test-func",
+         "component": "azure_servicebus",
+         "messaging.destination.name": "queue.1",
+         "messaging.operation": "send",
+         "messaging.system": "servicebus",
+         "network.destination.name": "localhost",
+         "span.kind": "producer"
+       },
+       "metrics": {
+         "_dd.top_level": 1
+       },
+       "duration": 7453458,
+       "start": 1749837120839785551
+     },
+        {
+          "name": "azure.functions.invoke",
+          "service": "test-func",
```

Comment:
If a user doesn't set `DD_SERVICE`, are we using the function name as the default service name for `azure.functions.invoke` spans?

Reply:
Correct, the function name from the `WEBSITE_SITE_NAME` environment variable will be used as the default service.

https://github.com/DataDog/dd-trace-py/blob/c7cd090668df7bde4c1d3bea66895ae949de77b2/ddtrace/settings/_config.py#L501-L502